snowdrop-core/src/Snowdrop/Core/Expand.hs

```diff
@@ -4,13 +4,12 @@
 {-# LANGUAGE ScopedTypeVariables     #-}
 {-# LANGUAGE UndecidableSuperClasses #-}
 
-module Snowdrop.Execution.Expand
+module Snowdrop.Core.Expand
```

Comment:
Now we will have `Snowdrop.Core.Expand` and `Snowdrop.Core.Expander`. This is counter-intuitive. Let's have two modules `Snowdrop.Core.Expander` -> `Snowdrop.Core.Expand.Type` and `Snowdrop.Core.Expand` 
 -> `Snowdrop.Core.Expand.Sequential`. And ofc  `Snowdrop.Core.Expand` with autoexporter pragma